### DISCUSSION TITLE
Any plans for turborepo integration?

### DISCUSSION THREAD

**Discussion by kratos-respawned** [danstepanov/create-expo-stack#97] [Ideas]
I'm encountering difficulties while trying to set up TurboRepo with Next.js and create-expo-app. Although I can use t3-turbo, it seems bit bloated/complex to me(skill issue )😥.
Are there any upcoming plans for a CLI that combines the strengths of TurboRepo and create-expo-stack? Alternatively, could someone please guide me on setting up expo-router with Tamagui in TurboRepo?

**Comment by ernestoresende** [comment]
@danstepanov This is what merging `create-cardinal-app` and `create-expo-stack` would look like 👀

Making something like this happen would be a colossal effort, but let me know if this is something you wanna consider talking about at some point.

**Comment by danstepanov** [comment]
@kratos-respawned We will be launching the paid starter (full stack) by the end of the month but it will use Expo Router for web. Create Expo Stack is more intended for rapid prototyping than actually being used to start a meaningful app/company. Would this address your concerns or are you deadset on using Next?

**Reply by kratos-respawned** [reply]
I will prefer nextjs for the web because It allows me to SSR the web app and the same backend server can be used for my expo app. I know expo router 3 comes with api routes but Nextjs dev xp is unmatched.

**Reply by danstepanov** [reply]
The paid starter is now launching with NextJS! https://nextth.ing
